pkg/volume/util/operationexecutor/operation_generator.go

```diff
@@ -1906,9 +1912,17 @@ func (og *operationGenerator) GenerateExpandInUseVolumeFunc(
 		var eventErr, detailedErr error
 		migrated := false
 
+		if currentSize.IsZero() || volumeToMount.PersistentVolumeSize.IsZero() {
```

Comment:
in what case this might happen?

Reply:
It should generally never happen but there is a possibility in some cases that size in ASOW is not populated and `GenerateExpandInUseVolumeFunc` got called.